addons/mail/static/src/models/message_view.js

```diff
@@ -488,6 +519,13 @@ registerModel({
             compute: '_computeExtraClass',
             default: '',
         }),
+        failureNotificationIconClassName: attr({
+            compute: '_computeFailureNotificationIconClassName',
+        }),
```

Comment:
Like the `failureNotificationIconLabel`, this could use `'fa fa-envelope-o'` as default value, and just return `clear()` in the compute.

Reply:
done